types/index.d.ts

```diff
@@ -289,10 +289,61 @@ declare namespace PhonegapPluginPush {
 		from?: string
 		notId?: string
 	}
-
+  /**
+   * Set Channel Variable,once created cannot be changed programmtically for importance and vibration
+   */
+  interface channel{
+    /**
+     * channel ID
+     */
+    id: string
+    /**
+     * camillebeaumont name and description fix
+     */
+    name: string 
+    /**
+     * camillebeaumont name and description fix,channel description
+     */
+    description: string
+    /**
+     * channel importance level
+     */
+    importance: integer
```

Comment:
The correct typescript type for numbers is `number`, we don't have specialized numeric types in typescript/javascript:
```suggestion
    importance: number
```

Reply:
Sorry about that! Not familiar with typescript